pytx3/setup.py

```diff
@@ -16,12 +16,13 @@
     description = f.read()
 
 with open(path.join(here, "version.txt"), encoding="utf-8") as f:
-    version = f.read()
+    version = f.read().strip()
```

Comment:
*Author note:* This is to address a User Warning when packaging related to the newline at the end of the file.